<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WatermarkForm</class>
 <widget class="QWidget" name="WatermarkForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Position</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0,0,0">
      <item>
       <widget class="QComboBox" name="comboPosition">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Top</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Top Left</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Top Right</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Left</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Right</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bottom</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bottom Left</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bottom Right</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Center</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Repeated</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item row="0" column="1">
         <widget class="QRadioButton" name="radioTop">
          <property name="toolTip">
           <string>Top</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="radioTopLeft">
          <property name="toolTip">
           <string>Top Left</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="radioTopRight">
          <property name="toolTip">
           <string>Top Right</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="radioLeft">
          <property name="toolTip">
           <string>Left</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="radioBottomLeft">
          <property name="toolTip">
           <string>Bottom Left</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QRadioButton" name="radioCenter">
          <property name="toolTip">
           <string>Center</string>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QRadioButton" name="radioBottom">
          <property name="toolTip">
           <string>Bottom</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QRadioButton" name="radioRight">
          <property name="toolTip">
           <string>Right</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QRadioButton" name="radioBottomRight">
          <property name="toolTip">
           <string>Bottom Right</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="radioRepeated">
        <property name="text">
         <string>&amp;Repeated</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Appearance</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Opacity</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSlider" name="sliderOpacity">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="sliderPosition">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinOpacity">
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Size</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="checkOriginalSize">
          <property name="text">
           <string>Keep original size</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSlider" name="sliderSize">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinSize">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Rotation</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="checkOriginalRotation">
          <property name="text">
           <string>Keep original rotation</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,0">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QDial" name="dialRotation">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>359</number>
          </property>
          <property name="value">
           <number>270</number>
          </property>
          <property name="sliderPosition">
           <number>270</number>
          </property>
          <property name="tracking">
           <bool>true</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="invertedControls">
           <bool>false</bool>
          </property>
          <property name="wrapping">
           <bool>true</bool>
          </property>
          <property name="notchesVisible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinRotation">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="wrapping">
           <bool>true</bool>
          </property>
          <property name="suffix">
           <string>°</string>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="maximum">
           <number>359</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Color</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="checkOriginalColor">
          <property name="text">
           <string>Keep original color</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="topMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="buttonPickColor">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sliderColorOpacity">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinColorOpacity">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Watermarks</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="currentRow">
         <number>-1</number>
        </property>
        <item>
         <property name="text">
          <string>No watermark</string>
         </property>
         <property name="icon">
          <iconset resource="../ressources/icons.qrc">
           <normaloff>:/icons/48px/no</normaloff>:/icons/48px/no</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="buttonAdd">
          <property name="text">
           <string>Add</string>
          </property>
          <property name="icon">
           <iconset resource="../ressources/icons.qrc">
            <normaloff>:/icons/48px/add</normaloff>:/icons/48px/add</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonRemove">
          <property name="text">
           <string>Remove</string>
          </property>
          <property name="icon">
           <iconset resource="../ressources/icons.qrc">
            <normaloff>:/icons/48px/remove</normaloff>:/icons/48px/remove</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ressources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
